File: python/move_zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        orig = len(nums)
        nums[:] = [x for x in nums if x != 0]
        diff = orig - len(nums)
        for i in range(diff):
            nums.append(0)

File: rust/move_zeroes.rs
impl Solution {
    pub fn move_zeroes(nums: &mut Vec<i32>) {
        let original: usize = nums.len();
        nums.retain(|&x| x != 0);
        let removed: usize = nums.len();
        let diff: usize = original - removed;
        for i in 0..diff{
            nums.push(0);
        } 
    }
}